mono/mini/aot-compiler.c

```diff
@@ -12423,6 +12503,7 @@ acfg_create (MonoAssembly *ass, guint32 opts)
 	acfg->method_to_cfg = g_hash_table_new (NULL, NULL);
 	acfg->token_info_hash = g_hash_table_new_full (NULL, NULL, NULL, NULL);
 	acfg->method_to_pinvoke_import = g_hash_table_new_full (NULL, NULL, NULL, g_free);
+	acfg->method_to_external_icall_symbol_name = g_hash_table_new_full (NULL, NULL, NULL, g_free);
 	acfg->image_hash = g_hash_table_new (NULL, NULL);
```

Comment:
This should simply be a global hash then.

Reply:
used the acfg to simplify ownership of the hash (alloc/free), but will be cached in global static during lifetime of acfg (similar to how we do with the llvm acfg), but agree, we could just move ownership out of acfg and put it in global hash, I guess we still want lifetime to be connected with the lifetime of acfg (or at least clean it)?

Reply:
Decided to keep current construction for now.